doc/sphinx-guides/source/installation/config.rst

```diff
@@ -2613,3 +2687,6 @@ To override the default list with Docs, Data, Code, and Workflow:
 To remove the override and go back to the default list:
 
 ``curl -X PUT -d '' http://localhost:8080/api/admin/settings/:FileCategories``
+
+.. |dvPageBlocks| image:: ./img/dvBrandingCustBlocks.png
```

Comment:
@j-n-c I'm working on a more complete review, but a quick question for you. How did you create this PNG file? What if we want to edit it in the future? Is it possible to get the source of the image? I'm thinking about an SVG file, for example. Something we can edit. Thanks.

Reply:
**nitpick**: This appears in the [preview](https://dataverse-guide--8638.org.readthedocs.build/en/8638/installation/config.html#branding-your-installation), but shouldn't the image be referenced at the place of use? Scrolling all the way down seems cumbersome.

Reply:
@pdurbin I have used [drawio](https://app.diagrams.net/) to create the image. You can download the software and run it offline or use the online editor.
Here is the editable file: [dv.zip](https://github.com/IQSS/dataverse/files/8609421/dv.zip)
.

Reply:
@j-n-c thanks! The "dv.drawio" is only 30 KB so I'll probably just commit it to my branch (I'm planning on making a pull request for you to review).

I was able to import it into https://app.diagrams.net without any trouble. Here's a screenshot:

![Screen Shot 2022-05-03 at 10 11 16 AM](https://user-images.githubusercontent.com/21006/166469894-50506d4e-71ef-4150-abf0-4e97f9bf649b.png)

@poikilotherm I guess it's personal preference. If you look at the README you'll see a long list of links at the bottom that are used throughout. I see this as similar. Put all that junk at the bottom.

Reply:
> nitpick: This appears in the [preview](https://dataverse-guide--8638.org.readthedocs.build/en/8638/installation/config.html#branding-your-installation), but shouldn't the image be referenced at the place of use? Scrolling all the way down (2000 lines) seems cumbersome.

@poikilotherm In https://github.com/ErykKul/dataverse/pull/1 I said "you were right". 😄 

People just want to add database settings to the bottom of the config page, as they should. Don't make me think!